build/Dockerfile

```diff
@@ -1,22 +0,0 @@
-# stage 1 building the code
-FROM golang:1.22 as builder
-
-ARG VERSION
-ARG SHORT_COMMIT
-ARG DATE
-
-COPY / /golangci
-WORKDIR /golangci
-RUN APP_VERSION=${VERSION#v} \
```

Comment:
I don't see the passing of these variables (`version`, `commit`, `date`) in the new approach.
How does `golangci-lint version` work now?

Reply:
UPD: I saw in `.goreleaser.yml`